### DISCUSSION TITLE
Does TImage32 support ScaleToFit?

### DISCUSSION THREAD

**Discussion by chicknsoup** [graphics32/graphics32#295] [General] [answered]
I have a TImage32 object loaded with an image of 350x350px, when resizing the Timage32 object to smaller dimensions (250x250 for example), the image will be cropped. Is the any way to make it fit the new size?

**Comment by andersmelander** [comment, marked as answer]
Use the [ScaleMode](https://graphics32.github.io/Docs/Units/GR32_Image/Classes/TCustomImage32/Properties/ScaleMode.htm) property.

**Reply by chicknsoup** [reply]
Thank you, my bad. I tried it once but somehow it did not work. Now it works perfectly.
